variables.tf

```diff
@@ -268,6 +268,22 @@ variable "logs_agent_additional_metadata" {
   default = []
 }
 
+variable "logs_agent_resources" {
+  description = "The resources configuration for cpu/memory"
+  type = object({
+    limits = object({
+      cpu               = string
+      memory            = string
+      ephemeral_storage = string
+    })
+    requests = object({
+      cpu               = string
+      memory            = string
+      ephemeral_storage = string
+    })
+  })
+}
```

Comment:
I think this should have a default value, otherwise the input becomes required for users to enter